tests/unit/mock_builder_test.ts

```diff
@@ -0,0 +1,62 @@
+import { asserts } from "../../deps.ts";
+import { MockBuilder } from "../../src/mock_builder.ts";
+import { Rhum } from "../../mod.ts";
+
+Deno.test({
+  name: "Unit | MockBuilder | constructor() | returns a MockBuilder object",
+  fn(): void {
+    const mock = new MockBuilder(TestObjectOne);
+    asserts.assertEquals(mock.constructor.name, "MockBuilder");
+  },
+});
+
+Deno.test({
+  name: "Unit | MockBuilder | create() | creates mock without constructor args",
+  fn(): void {
+    const mock = new MockBuilder(TestObjectTwo)
+      .create();
+    asserts.assertEquals(mock.name, undefined);
+  },
+});
+
+Deno.test({
+  name: "Unit | MockBuilder | create() | creates mock with constrcutor args",
+  fn(): void {
+    const mock = new MockBuilder(TestObjectTwo)
+      .withConstructorArgs("some name")
+      .create();
+    asserts.assertEquals(mock.name, "some name");
```

Comment:
why `name` and not `args` or `arguments`?
would more than one be `.withConstructorArgs("some", "name")` output `asserts.assertEquals(mock.name, ["some", "name"]);`?

Reply:
its `name` because the constructor sets `"some name"` to the `name` property. it's just asserting that the constructor did what it was supposed to

> would more than one be .withConstructorArgs("some", "name") output asserts.assertEquals(mock.name, ["some", "name"]);?

maybe, but the constructor for that specific mock only allows 1 argument

Reply:
Sara are you still looking at this?